ios/shapeshift.xcodeproj/project.pbxproj

```diff
@@ -640,7 +640,7 @@
 				CURRENT_PROJECT_VERSION = 318;
 				DEVELOPMENT_TEAM = 7882V35EPB;
 				INFOPLIST_FILE = shapeshift/Info.plist;
-				INFOPLIST_KEY_CFBundleDisplayName = ShapeShift;
+				INFOPLIST_KEY_CFBundleDisplayName = ShapeShifte;
```

Comment:
uh oh @NeOMakinG

Reply:
damn xcode is trolling there, I added this `e` to trigger multiple changes, then changed it back but seems like it didn't change it there

I fixed it